codeHF/workflows.yml

```diff
@@ -482,15 +482,21 @@ workflows:
   o2-analysis-pid-tpc-full:
     dependencies: [o2-analysis-pid-tpc-base, o2-analysis-timestamp]
 
-  o2-analysis-pid-tof-base:
+  o2-analysis-pid-tof-base_run2:
+    executable: o2-analysis-pid-tof-base
     dependencies: o2-analysis-event-selection
 
-  o2-analysis-pid-tof-full_run2: &tof_full
+  o2-analysis-pid-tof-base_run3:
+    executable: o2-analysis-pid-tof-base
+    dependencies: [o2-analysis-event-selection, o2-analysis-ft0-corrected-table]
```

Comment:
Add the `o2-analysis-ft0-corrected-table` workflow at the end of the helper tasks block.

Reply:
I think this also needs to replace `processNoFT0` with `processFT0` in `config_tasks.sh`. Otherwise the `ft0-corrected-table` workflow is not needed. Please check what your current JSON settings are because if you don't have `processFT0 = true`, I don't see why you need the FT0 workflow.

Reply:
Luigi's setting on Hyperloop had processFT0=true. I needed to modify config_task.sh so as not to have processFT0 set automatically to false. See the question 2. of my general PR comment.

I am not knowledgeable in what impact switching FT0 has on analysis... Luigi will surely know more why he choose such settings.

Reply:
Yes, I am referring to the same lines as you link in your question 2. Please include that change in `config_tasks.sh`.